src/handlebars/jmc.handlebars

```diff
@@ -51,21 +51,51 @@
         <td class="first_col">Windows</td>
         <td>8.0.0 Snapshot</td>
         <td>
-          <a href="https://ci.adoptopenjdk.net/view/JMC/job/jmc-build/job/master/lastSuccessfulBuild/artifact/target/products/org.openjdk.jmc-8.0.0-SNAPSHOT-win32.win32.x86_64.zip">org.openjdk.jmc-8.0.0-SNAPSHOT-win32.win32.x86_64.zip</a>
+          <a href="https://ci.adoptopenjdk.net/view/JMC/job/jmc-build/job/8.x.x/lastSuccessfulBuild/artifact/target/products/org.openjdk.jmc-8.0.0-SNAPSHOT-win32.win32.x86_64.zip">org.openjdk.jmc-8.0.0-SNAPSHOT-win32.win32.x86_64.zip</a>
         </td>
       </tr>
       <tr>
         <td class="first_col">Mac</td>
         <td>8.0.0 Snapshot</td>
         <td>
-          <a href="https://ci.adoptopenjdk.net/view/JMC/job/jmc-build/job/master/lastSuccessfulBuild/artifact/target/products/org.openjdk.jmc-8.0.0-SNAPSHOT-macosx.cocoa.x86_64.tar.gz">org.openjdk.jmc-8.0.0-SNAPSHOT-macosx.cocoa.x86_64.tar.gz</a>
+          <a href="https://ci.adoptopenjdk.net/view/JMC/job/jmc-build/job/8.x.x/lastSuccessfulBuild/artifact/target/products/org.openjdk.jmc-8.0.0-SNAPSHOT-macosx.cocoa.x86_64.tar.gz">org.openjdk.jmc-8.0.0-SNAPSHOT-macosx.cocoa.x86_64.tar.gz</a>
         </td>
       </tr>
       <tr>
         <td class="first_col">Linux</td>
         <td>8.0.0 Snapshot</td>
        <td>
-          <a href="https://ci.adoptopenjdk.net/view/JMC/job/jmc-build/job/master/lastSuccessfulBuild/artifact/target/products/org.openjdk.jmc-8.0.0-SNAPSHOT-linux.gtk.x86_64.tar.gz">org.openjdk.jmc-8.0.0-SNAPSHOT-linux.gtk.x86_64.tar.gz</a>
+          <a href="https://ci.adoptopenjdk.net/view/JMC/job/jmc-build/job/8.x.x/lastSuccessfulBuild/artifact/target/products/org.openjdk.jmc-8.0.0-SNAPSHOT-linux.gtk.x86_64.tar.gz">org.openjdk.jmc-8.0.0-SNAPSHOT-linux.gtk.x86_64.tar.gz</a>
+        </td>
+      </tr>
+    </table>
+
+    <h2 class="medium-title">Preview JDK Mission Control 8.1.0</h2>
+    <table id="release">
+      <tr>
+        <th>System</th>
+        <th>Version</th>
+        <th>Download</th>
+      </tr>
+      <tr>
+        <td class="first_col">Windows</td>
+        <td>8.1.0 Snapshot</td>
+        <td>
+          <a href="https://ci.adoptopenjdk.net/view/JMC/job/jmc-build/job/master/lastSuccessfulBuild/artifact/target/products/org.openjdk.jmc-8.1.0-SNAPSHOT-win32.win32.x86_64.zip">org.openjdk.jmc-8.0.0-SNAPSHOT-win32.win32.x86_64.zip</a>
```

Comment:
the description is wrong (8.0.0) - repeated below as well.

Reply:
Ah yes of course 🤦 , thanks for the feedback, I just fixed it !